CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 8.9.1
+
+This release fixes a publishing issue.
```

Comment:
So I would move all the changes from `8.9.0` here, and then add a note to `8.9.0` about using `8.9.1` instead.

Reply:
👍 updated